app.js

```diff
@@ -26,7 +27,7 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use('/', indexRouter, gamesRouter, studioRouter);
 
 // catch 404 and forward to error handler
-app.use(function(req, res, next) {
+app.use('*', function(req, res, next) {
```

Comment:
By default, the location of this middleware will catch any unhandled requests. What does adding in a * glob do for us?

Reply:
Nothing